### ISSUE TITLE
[Accessibility][Usability]A11y_VSCode_DebugConsole_ScreenReader: NVDA does not read the results at the time when user runs the command.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52404]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version: Win10
OS Version : HighSierra10.13.5 

**Additional Details:**
ScreenReader: VoiceOver
NVDA Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while VoiceOver/NVDA is ON.
2. Click on View menu item--> Click on Debug Console.

**Actual:**
NVDA does not read the results at the time when user runs the command.If user wants to know the output or result, he has to navigate to the treeview then he would be able to read the results using arrow keys.

**Expected:**
ScreenReader should read the result at the time when user runs any command, If he wants to see the results of previous commands then he can navigate to treeview and see the results.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Everytime user will have to navigate to the treeview to see the result.

Attachment for Reference: 
[A11y VSCode  DebugConsole NVDA not reading results.pptx](https://github.com/Microsoft/vscode/files/2118220/A11y.VSCode.DebugConsole.NVDA.not.reading.results.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Disagree. If we read this out everytime a debug console is opened it would be too much information for the user. Especaily since we automatically open the debug console at times.
Closing as designed
